pandas/__init__.py

```diff
@@ -83,6 +83,7 @@
 from ._version import get_versions
 v = get_versions()
 __version__ = v.get('closest-tag', v['version'])
+__git_version__ = v['full-revisionid']
```

Comment:
this won’t work for released version
use .get

pls add a test for this

Reply:
Adding test to check if it returns valid sha should be okay?